jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/JibPlugin.java

```diff
@@ -106,7 +106,9 @@ public void apply(Project project) {
           try {
             War warTask = GradleProjectProperties.getWarTask(project);
             Task dependsOnTask;
-            if (warTask != null) {
+            if (warTask != null
+                && (jibExtension.getPackagingOverride() == null
```

Comment:
Can this use `TaskCommon.isWarContainerization` or at least share the same logic? If `jib.packagingOverride` system property is set, this would not be affected.